### PR TITLE
[KYUUBI #2745] [Subtask] DorisSQLEngine - GetTables Operation

### PR DESCRIPTION
**Pull request by zhaomin1423**
### _Why are the changes needed?_

close https://github.com/apache/incubator-kyuubi/issues/2745#issue-1248428722

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by zhaomin1423** [comment]
please help me review it. @yaooqinn @pan3793 @ulysses-you

**Review by yaooqinn — APPROVED** [review]

**Comment by zhaomin1423** [comment]
The ci failed because of https://github.com/apache/incubator-kyuubi/pull/2816

**Comment by zhaomin1423** [comment]
The doris container exception don't exist, there is a failure about LADP.

**Comment by ulysses-you** [comment]
thanks, merging to master

File: kyuubi-common/src/main/scala/org/apache/kyuubi/operation/meta/ResultSetSchemaConstant.scala [hunk 1]
```diff
@@ -37,6 +37,12 @@ object ResultSetSchemaConstant {
    */
   final val TABLE_SCHEM = "TABLE_SCHEM"
```
Comment (turboFei) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/operation/meta/ResultSetSchemaConstant.scala:38]: typo and duplicated with below TABLE_SCHEMA.

cc @lightning-L

**Pull request closed without merging**